manifold_airflow_dags/tupress_ingest_dag.py

```diff
@@ -84,6 +87,19 @@ def calculate_file_to_move(**context):
     dag=DAG
 )
 
+ingest_deltas_bash = """
+sudo su - { TUPRESS_USER_NAME } bash -c \
+ "cd /var/www/tupress &&\
+ RAILS_ENV=production bundle exec rake db:seed:run_updates[\"%s\"]"
+""" % f"{ TUPRESS_WEB_PATH }/%s" % "{{ ti.xcom_pull(task_ids='get_file_to_transfer') }}"
+
```

Comment:
I don't think `TUPRESS_USER_NAME` will not get interpolated.

```
>>> foo = "hello"
>>>
>>> bar = """ hello {foo} """
>>>
>>> bar
' hello {foo} '
```

Reply:
I will template as indicated below.